endoreg_db/management/commands/import_video.py

```
@@ -32,6 +37,10 @@
     load_ai_model_label_data,
     load_ai_model_data,
     load_default_ai_model
+    
+)
+from endoreg_db.helpers.default_objects import (
+    get_latest_segmentation_model
```

Ruff (0.11.9)

Redefinition of unused `get_latest_segmentation_model` from line 24

(F811)